web/src/compose_validate.ts

```diff
@@ -580,6 +616,9 @@ export function validation_error(error_type, stream_name) {
                 return false;
             }
             const sub = stream_data.get_sub(stream_name);
+            if (!sub) {
+                return false;
+            }
```

Comment:
This is the wrong conversion; you should instead treat `!sub` as another case where `stream_data.can_toggle_subscription(sub)` should be `false`.